packages/terra-modal/src/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: $terra-white;
```

Comment:
I think we want to keep left, right, top, and bottom as 10px instead of rem. If the user updates the font size in their browser, we probably want the area around the modal dialog to stay consistent.
**Large Font**
![largefont](https://cloud.githubusercontent.com/assets/8146159/25296392/af190d36-26ad-11e7-98c9-acbe1cec829d.png)
**Small Font**
![smallfont](https://cloud.githubusercontent.com/assets/8146159/25296391/af18af4e-26ad-11e7-8047-d80b3cfda759.png)